src/Context/CredentialsOAuthContext.php

```diff
@@ -62,6 +62,11 @@ public function intoUserIdContext(?string $targetCustomerId = null): UserIdOAuth
         return new UserIdOAuthContext($this->clientId, $this->clientSecret, $targetCustomerId);
     }
 
+    public function intoClientTokenContext(): ClientTokenOAuthContext
+    {
+        return new ClientTokenOAuthContext($this->clientId, $this->clientSecret);
```

Comment:
```suggestion
        return new ClientTokenOAuthContext($this->clientId,  #[\SensitiveParameter] $this->clientSecret);
```

Idk, maybe a good fit

Reply:
Attributes cannot be added while passing values to other functions, but rather on the parameters themselves, see [here](https://github.com/shopware/paypal-sdk/blob/e4d756a826a354d8d8c563c48c267252d5d677b4/src/Context/CredentialsOAuthContext.php#L19). Due to `ClientTokenOAuthContext` extending `CredentialsOAuthContext`, the `$clientSecret` parameter already has a sensitive parameter attribute.